index.html

```diff
@@ -3756,6 +3756,12 @@ <h2>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&gt; ( did-resolution-metadata, did-document-stream, did-document-metadata )
         </code></p>
 
+        <p>
+The first <code>resolve</code> form returns the DID document in its abstract form
+without ever having to deserialize or parse.
+The second <code>resolveRepresentation</code> form returns a 
+byte stream still formatted in the corresponding representation.
```

Comment:
I think my suggested changes still capture the essence of your text, but I think it reads more smoothly.

```suggestion
The <code>resolve</code> function returns the <a>DID document</a> in its
abstract form.
The <code>resolveRepresentation</code> function returns a byte stream of the
<a>DID Document</a> formatted in the corresponding representation.
```

Reply:
Thank you. updated